doc/source/changes/version_0_22.rst.inc

```diff
@@ -190,6 +190,8 @@ Miscellaneous improvements
 
   will autocomplete to `s.a_nice_test_array` and `s['a_<tab>` will be completed to `s['a_nice_test_array`
 
+* tests: replaced nose by pytest (closes :issue:`232`)
```

Comment:
Unsure we should mention this. So far, the changelog has been user-oriented and I do not think any of our current users cares about this.